### ISSUE TITLE
Disable for div with id "tipsy"

### ISSUE THREAD

**Issue by Lordmau5** [popcorncolonel/Chrome-Extensions#7]

**Comment by popcorncolonel** [comment]
What if there are other sites that use this framework that users would want emotes to be replaced in?

**Comment by Lordmau5** [comment]
Hmm... I would only think about implementing it into the website-blacklist...

I tried to blacklist twitch.tv so it would shouldn't show up anymore, but it still did :/

**Comment by popcorncolonel** [comment]
Yeah, that's because the blacklist looks at location.hostname rather than the host of the iframe it's in.
